specs.json

```
@@ -516,7 +516,14 @@
     }
   },
   "https://wicg.github.io/css-parser-api/",
-  "https://wicg.github.io/custom-state-pseudo-class/",
+  {
+    "url": "https://wicg.github.io/custom-state-pseudo-class/",
+    "standing": "discontinued",
+    "obsoletedBy": [
+      "html",
+      "selectors-4"
```

```suggestion
      "selectors-5"
```
As annoying as it is, I think the right solution is not to merge this until the CSS bits actually surface in a new css-selectors-5; at least, here is my reasoning:
* neither the spec nor the repo are marked as discontinued
* clearly the HTML spec has taken over the IDL bit, but this is properly taken care of by your patch in https://github.com/w3c/webref/pull/1119/files
* you can't quite make sense of the feature without what's described in the spec until it gets [moved into the new CSS selector 5 spec ](https://github.com/w3c/csswg-drafts/pull/8213/files)
* referencing css-selectors-4 when it doesn't actually describe the feature doesn't really help anyone

---

Works for me. Let's park this for now!